### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by LOctavio**
For this milestone I did the following;

- Implement the design from the sneak peek wireframe
image
image
image
image

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,11 @@
...
+<%= render "shared/user", user: @user %>
+<% @posts.each do |post| %>
+  <%= link_to user_post_path(@user, post) do %>
+    <div class="post-and-comments">
+      <%= render "shared/post", post: post %>
+      <div class="box">
+        <%= render "shared/comments", post: post, comments: post.most_recent_comments %>
+      </div>
+    </div>
+  <% end %>
```
Comment (DeliceLydia) [inline app/views/posts/index.html.erb:10]: - [ ] Please use `most_recent_post` method to render the user's 3 most recent posts currently when I check users posts I get all posts and according to the requirements we should see only 3recents posts

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,13 @@
...
+  <span><%= @user.bio %></span>
+</div>
+<% @posts.each do |post| %>
+  <%= link_to user_post_path(@user, post) do %>
+    <%= render "shared/post", post: post %>
+  <% end %>
+<% end %>
+<div class="button-container">
+  <%= button_to "See all posts", user_posts_path(@user),:class => "posts-button", :method => :get %>
+</div>
```
Comment (DeliceLydia) [inline app/views/users/show.html.erb:13]: - [ ] When I click on this link I should be able to see a list of all user's posts but currently am getting this error please fix that
link

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @LOctavio, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by henatan99 — APPROVED** [review]
Hi @LOctavio , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**